ngrinder-controller/src/main/java/org/ngrinder/perftest/controller/PerfTestApiController.java

```diff
@@ -13,6 +13,7 @@
  */
 package org.ngrinder.perftest.controller;
 
+import lombok.RequiredArgsConstructor;
```

Comment:
Whenever u change the import order is changed.

@imbyungjun  @donggyu04 do you guys same style?

Reply:
We'll check IDE settings.